CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-*no unreleased changes*
+* Added new branch of CANQL for querying all patient not just child patients (#29)
```

Comment:
Not sure I'm clear on what the breaking change is - if this is adding a `results.subject` key to the DIR, but otherwise leaving the output of existing queries unchanged, I think this is just additive?

Reply:
Chatted through this, and I agree with you.